hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -108,14 +108,17 @@
   private transient Timer.Context indexTimer = null;
 
   /**
-   *
+   * @param jsc Java Spark Context
+   * @param clientConfig instance of HoodieWriteConfig
    */
-  public HoodieWriteClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig) throws Exception {
+  public HoodieWriteClient(JavaSparkContext jsc, HoodieWriteConfig clientConfig) {
     this(jsc, clientConfig, false);
   }
 
   /**
-   *
+   * @param jsc Java Spark Context
```

Comment:
if possible add a line about instantiation and what this constructor is used for compared to other constructors.

Reply:
nit: `Constructor function` seems little obvious.. Lets change both to 

`Creates a write client, without cleaning up failed/inflight commits` ..

 In general, let's just describe important behavioral things...